modules/lm-coursier/src/main/scala/lmcoursier/CoursierConfiguration.scala

```diff
@@ -54,9 +56,29 @@ import scala.concurrent.duration.Duration
   @since
   sbtClassifiers: Boolean = false,
   @since
-  providedInCompile: Boolean = false // unused, kept for binary compatibility
+  providedInCompile: Boolean = false, // unused, kept for binary compatibility
+  @since
+  protocolHandlerDependencies: Seq[ModuleID] = Vector.empty,
 ) {
 
+  val classLoaders: Seq[ClassLoader] = {
+    import coursier.{Dependency, Fetch}
+    import coursier.core.{Module, Organization, ModuleName}
+
+    val dependencies = protocolHandlerDependencies.map(dep => 
+      Dependency(
+        module = Module(
+          organization = Organization(dep.organization),
+          name = ModuleName(dep.name + "_2.12"),
+          attributes = Map()
+        ),
+        version = dep.revision
+      )
+    )
```

Comment:
There is probably a better way to do this. I'm not using all the sbt-coursier configurations available here.

Reply:
Yep, you can use `lmcoursier.FromSbt.dependencies`, and just discard the `Configuration`s in the returned seq. `scala.util.Properties.versionNumberString` should be passed as Scala version I guess, and `sbt.librarymanagement.CrossVersion.binaryScalaVersion` can be used to compute the binary Scala version.

Reply:
You might also want to allow users to add repositories. `lmcoursier.internal.Resolvers.repository` allows to go from sbt's `Resolver` to `coursier.Repository`.

Ideally, users should be allowed to pass a `lmcoursier.CoursierConfiguration`, then resolution should be done via `lmcoursier.CoursierDependencyResolution`, but it may not be worth the hassle...

Reply:
I ended up going with the resolution with `lmcoursier.CoursierDependencyResolution` It was super easy.